darwin/Platform.h

```diff
@@ -31,6 +31,8 @@ extern const unsigned int Platform_numberOfSignals;
 
 extern const MeterClass* const Platform_meterTypes[];
 
+extern long Platform_clockTicksPerSec;
```

Comment:
Probably something for another PR, but if multiple platforms have this (Darwin + Linux, but basically *BSD too), this should most likely go into a global platform header (cf. #306) for some central interface for such universally needed information.

Reply:
Agreed. See also: https://github.com/htop-dev/htop/issues/384

Reply:
This comment was mostly about this declaration itself, which is probably better suited to have in some platform-independent header and not with every platform.